charts/prometheus-ipmi-exporter/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: prometheus-ipmi-exporter
+description: This is an IPMI exporter for Prometheus.
+
+type: application
+
+version: 0.1.0
+
+appVersion: "1.6.1"
```

Comment:
@lexfrei 
Thanks for the contribution. Would you mind to add some maintainer(s), e.g. yourself? A chart without maintainer(s) would be hard to keep it up.

Reply:
Im fine to maintain this chart. Please, give me a link to some guides about how to do this in prometheus community :)